### PR TITLE
[TOSA] Reformat function names to match pep 8 convention

### PR DESCRIPTION
**Pull request by Jerry-Ge**
- reformatted utility function signatures to match pep 8 convention

This PR is created to fix this issue https://github.com/pytorch/executorch/issues/1370

### CODE REVIEW

File: backends/arm/arm_backend.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Arm Limited and/or its affiliates.
+# Copyright 2023-2024 Arm Limited and/or its affiliates.
```
Comment (digantdesai) [inline backends/arm/arm_backend.py:1]: Happy new year to you too :)

**Review by digantdesai — APPROVED** [review]
LGTM, thanks. I am assuming no functional changes.

File: backends/arm/arm_backend.py (hunk 1 above)
Comment (Jerry-Ge) [reply backends/arm/arm_backend.py:1]: thanks Digant! Yes, no functional changes. And it's ready to be merged. : )

**Comment by Jerry-Ge** [comment]
Good morning @digantdesai , this PR has been approved for 3 days and haven't been merged. Is there anything prohibiting the merge somewhere? thanks : )

**Comment by digantdesai** [comment]
Sorry got busy with performance review cycle, importing now.

**Pull request closed without merging**